docs/guides/src/main/server/hostname.adoc

```diff
@@ -64,6 +64,17 @@ To perform this action, you can override the hostname path to use the path defin
 The `hostname-path` configuration takes effect when a reverse proxy is enabled.
 For details, see the <@links.server id="reverseproxy"/> Guide.
 
+== Accessing Keycloak in production mode using HTTP
+Keycloak follows the "secure by design" principle, so it is absolutely not recommmended to access Keycloak without proper transport encryption, as this opens up multiple attack vectors.
```

Comment:
```suggestion
When a `hostname` is set and the server is running in production mode, all the URLs generated by the server are going to use the `HTTPS` scheme. If you are not setting up TLS you might run into issues because some URLs generated by the server won't work.

Keycloak follows the "secure by design" principle, so it is absolutely not recommended to access Keycloak without proper transport encryption, as this opens up multiple attack vectors.
```